doc/nrf/libraries/bin/lwm2m_carrier/app_integration.rst

```diff
@@ -145,39 +146,50 @@ Following are some of the general Kconfig options that you can configure:
 Server options
 ==============
 
-Following are some of the server Kconfig options that you can configure:
+Following are some of the server Kconfig options that you can configure.
+See the ``Enabled Carriers`` under :ref:`general_options_lwm2m` for when the option is relevant.
```

Comment:
Is this a section? If so, linking would be best.

Reply:
Not a section. But perhaps it should be. I will consider how to group these across Kconfig/rst/.h file